src/sdk/pynni/nni/_graph_utils.py

```diff
@@ -199,6 +203,8 @@ def __init__(self, name, unique_name, node_type, op_type, node_cpps, inputs=None
             All the inputs of this node, each element is debugName of one input
         outputs: list of str
             All the outputs of this node, each element is debugName of one output
+        key_node: torch._C.Node
+            The key node of this NodePyGroup.
```

Comment:
what is the meaning of key node?

Reply:
Key nodes are the nodes that should not be merged into other nodes. In the past, we only take the `aten::` nodes as the important(key) nodes. In this pr, we also take the list/tuple unpack nodes as the key nodes.